src/com/google/javascript/jscomp/CodeGenerator.java

```diff
@@ -1084,8 +1084,8 @@ protected void add(Node n, Context context) {
             // Object literal value.
             checkState(
                 !isInClass, "initializers should only exist in object literals, not classes");
-            cc.addOp(":", false);
-            add(initializer);
+            cc.add(":");
+            addExpr(initializer, 1, Context.OTHER);
```

Comment:
Could you add a comment here?
```java
// Must use addExpr() with a priority of 1, because a comma expression here would cause a
// syntax error within the object literal.
```